daisy/workflow/step_wait_for_instances_signal.go

```diff
@@ -92,49 +92,49 @@ func waitForSerialOutput(w *Workflow, name string, port int64, success, failure
 }
 
 func (w *WaitForInstancesSignal) populate(ctx context.Context, s *Step) error {
-	for _, ws := range *w {
+	for i, ws := range *w {
 		if ws.Interval == "" {
 			ws.Interval = defaultInterval
 		}
-		interval, err := time.ParseDuration(ws.Interval)
+		var err error
+		ws.interval, err = time.ParseDuration(ws.Interval)
 		if err != nil {
 			return err
 		}
-		ws.interval = interval
+		(*w)[i] = ws
```

Comment:
Why not convert WaitForInstancesSignal []InstancesSignal to []*InstancesSignal? That's what I've been doing in the other cases.